angular-ngrx-scss/src/app/state/repository/repository.selectors.ts

```diff
@@ -1,26 +1,26 @@
 import { createFeatureSelector, createSelector } from '@ngrx/store';
-import { RepoState } from './repository.state';
+import { RepositoryState } from './repository.state';
 
-export const repositoryFeatureKey = 'repo';
+export const repositoryFeatureKey = 'repository';
 export const selectRepositoryState =
-  createFeatureSelector<RepoState>(repositoryFeatureKey);
+  createFeatureSelector<RepositoryState>(repositoryFeatureKey);
 
 export const selectedRepository = createSelector(
   selectRepositoryState,
-  (state: RepoState) => state,
+  (state: RepositoryState) => state,
 );
 
 export const selectCurrentlySelectedFile = createSelector(
   selectRepositoryState,
-  (state: RepoState) => state.selectedFile,
+  (state: RepositoryState) => state.selectedFile,
```

Comment:
we don't need to explicitly type the `state` parameter. `createSelector` is strictly typed and will infer that correctly from the return type of `selectRepositoryState`